content/en/user-guide/aws/lambda/index.md

```diff
@@ -119,6 +109,7 @@ LocalStack provides various tools to help you develop, debug, and test your AWS
 
 * **Hot reloading**: With Lambda hot reloading, you can continuously apply code changes to your Lambda functions without needing to redeploy them manually. To learn more about how to use hot reloading with LocalStack, check out our [hot reloading documentation]({{< ref "hot-reloading" >}}).
 * **Remote debugging**: LocalStack's remote debugging functionality allows you to attach a debugger to your Lambda function using your preferred IDE. To get started with remote debugging in LocalStack, see our [debugging documentation]({{< ref "debugging" >}}).
+* **Lambda VS Code Extension**: LocalStack's Lambda VS Code Extension supports deploying and invoking Python Lambda functions through AWS SAM or AWS CloudFormation. To get started with the Lambda VS Code Extension, see our [Lambda VS Code Extension documentation]({{< ref "user-guide/tools/lambda-vscode-extension" >}}).
```

Comment:
https://github.com/localstack/docs/actions/runs/6523369380/job/17713946746#step:5:26
`ERROR 2023/10/15 10:00:38 [en] REF_NOT_FOUND: Ref "user-guide/tools/lambda-vscode-extension": "/home/runner/work/docs/docs/content/en/user-guide/aws/lambda/index.md:112:265": page not found`

Adding the `lambda-tools` fixes the build error